### ISSUE TITLE
Testbed `look_at()` method doesn't seem to center the camera at the location given

### ISSUE THREAD

**Issue by MJohnson459** [rustsim/nphysics#243]
I've been following the tutorial at https://nphysics.org/nphysics_testbed/ with some minor alterations but I can't understand what the `look_at()` method is doing. I have created a window and set the look_at position to the center of the view window (in theory) but this is not what appears.

Example code:
```rust
const WIDTH: f32 = 60.0;
const HEIGHT: f32 = 40.0;

fn main() {
    let mechanical_world = DefaultMechanicalWorld::new(Vector2::new(0.0, 0.0));
    let geometrical_world = DefaultGeometricalWorld::new();

    let mut bodies = DefaultBodySet::new();
    let mut colliders = DefaultColliderSet::new();
    let joint_constraints = DefaultJointConstraintSet::new();
    let force_generators = DefaultForceGeneratorSet::new();

    // Build a static ground body and add it to the body set.
    let ground_handle = bodies.insert(Ground::new());

   /// .. Spawn some objects at center and axis ...

    let mut testbed = Testbed::new_empty();

    // Tell the testbed your ground has an handle equal to `ground_handle`.
    // This will make it be drawn in gray.
    testbed.set_ground_handle(Some(ground_handle));
    // Provide to the testbed all the components of our physics simulation.
    testbed.set_world(
        mechanical_world,
        geometrical_world,
        bodies,
        colliders,
        joint_constraints,
        force_generators,
    );
    // Adjust the initial camera pose.
    let center = Point2::new(WIDTH / 2.0, HEIGHT / 2.0);
    println!("Center = {}", center);
    testbed.look_at(center, 20.0);
    testbed.add_callback(tick);
    testbed.run();
}
```
When run I get this window which is not the center I put in:
![Screenshot from 2019-12-19 12-37-53](https://user-images.githubusercontent.com/1353401/71174229-9d52b400-225c-11ea-9240-6daede89ba6e.png)

Is there a bug in the `look_at()` method or is there something we could add to the documentation to make it clearer what the expected behavior is?

**Comment by sebcrozet** [comment]
Hi!

Perhaps explaining how the projection is setup will help understand how `look_at` works.

If you have a zoom factor equal to 1, the coordinate system of the 2D projection matrix is such that:

- The point `(0, 0)` is at the center of the window.
- The point `(W/2, H/2)` is at the top-right corner of the window.
- The point `(-W/2, -H/2)` is at the bottom-left of corner the window.

where `W` and `H` are the width and height of the window.
So if you have a zoom factor equal to 20 as in your example, simply divide those values by 20, e.g., the point `(W/40, H/40)` will be at the top-right corner of the window.

In any case, no matter the zoom factor, if you want to position the camera such that objects at the coordinate `(0, 0)` lie in the center of the window, then you should simply `look_at` the point `(0, 0)`.

**Comment by MJohnson459** [comment]
Thanks for the explanation, I wasn't really sure what the zoom factor was. I think my problem is still a potential bug though as in the example I set the `look_at` position to be positive `(30, 20)` which I would expect to make the window center on that position instead of `(0,0)`. However it seems to be doing something different (perhaps centering on `(-30, -20)`?).

The image isn't very clear but that line in the top half is a visible x-axis and there are balls spawned at `(30,20)` which should be visible.

```rust
    let center = Point2::new(WIDTH / 2.0, HEIGHT / 2.0);
    println!("Center = {}", center);
    testbed.look_at(center, 20.0);
```

**Comment by MJohnson459** [comment]
This should be fixed in `nphysics` as of 0.14.0 as it updates `kiss3d` to 0.23.0 which contains the fix.
